Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM python:3.10
+
+RUN mkdir /blood_donation_platform
+
+WORKDIR /blood_donation_platform
+
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+RUN pip install --upgrade pip
+
+COPY . /blood_donation_platform
```

Comment:
This should be moved above under the workdir command to make the commands more coherant in the Dockerfile.